schema/training-module.js

```diff
@@ -0,0 +1,20 @@
+const { UUID, UUIDV1, STRING, DATEONLY, TEXT, BOOLEAN } = require('sequelize');
+
+module.exports = db => {
+
+  const TrainingModule = db.define('trainingModule', {
+    id: { type: UUID, defaultValue: UUIDV1, primaryKey: true },
+    migrated_id: STRING,
+    number: STRING,
```

Comment:
I think `number` is a really bad name for this field - insofar as none of the data actually contains a number for this field. I wouldn't inherit it from ASPeL and would call it `module` or something like that.

Reply:
👍

Reply:
what are your thoughts on the last 2 fields? exemption and exemption description? I guess we won't know if they are populated until we see real data?